UPGRADE.rst

```diff
@@ -49,6 +49,51 @@ returned by the Client-Server API:
     # configured on port 443.
     curl -kv https://<host.name>/_matrix/client/versions 2>&1 | grep "Server:"
 
+Upgrading to v1.0
+=================
+
+Validation of TLS certificates Synapse v1.0 is the first release to enforce
+validation of TLS certificates for the federation API. It is therefore
+essential that your certificates are correctly configured. See the `FAQ
+<docs/MSC1711_certificates_FAQ.md>`_ for more information.
+
+Note, v1.0 installations will also no longer be able to federate with servers
+that have not correctly configured their certificates.
+
+In rare cases, it may be desirable to disable certificate checking: for
+example, it might be essential to be able to federate with a given legacy
+server in a closed federation. This can be done in one of two ways:- *
+Configure the global switch ``federation_verify_certificates`` to ``false``. *
+Configure a whitelist of server domains to trust via
+``federation_certificate_verification_whitelist``.
+
+See the `sample configuration file
+<https://github.com/matrix-org/synapse/blob/master/docs/sample_config.yaml>`_
+for more details on these settings. Password reset emails When a user requests
```

Comment:
Not sure if `Password reset emails` is supposed to be a title or some text that got in the way here.

Reply:
done